rmgpy/reaction.py

```diff
@@ -1144,6 +1144,121 @@ def ensure_species(self, reactant_resonance=False, product_resonance=True):
         except AttributeError:
             pass
 
+    def check_collision_limit_violation(self, t_min, t_max, p_min, p_max):
+        """
+        Warn if a core reaction violates the collision limit rate in either the forward or reverse direction
+        at the relevant extreme T/P conditions. Assuming a monotonic behaviour of the kinetics.
+        Returns a list with the reaction object and the direction in which the violation was detected.
+        """
+        conditions = [[t_min, p_min]]
+        if t_min != t_max:
+            conditions.append([t_max, p_min])
+        pdep_conditions = conditions
+        if p_max != p_min:
+            pdep_conditions.append([t_min, p_max])
+            if t_min != t_max:
+                pdep_conditions.append([t_max, p_max])
+        logging.debug("Checking whether reaction {0} violates the collision rate limit...".format(self))
+        f_violator = None
+        r_violator = None
+        kf_list = []
+        kr_list = []
+        if len(self.reactants) == 2:
+            collision_limit = []
+            for condition in conditions:
+                try:
+                    collision_limit.append(self.calculate_coll_limit(temp=condition[0], reverse=False))
+                except ValueError:
```

Comment:
Under what circumstances does this try statement throw a ValueError? Is it something that is okay to ignore?

Reply:
When I tested this branch by inventing ridiculous rates, I got a `ValueError` from `calculate_coll_limit()`. It was a while ago, so I don't remember exactly what calculation caused it to crash (perhaps either `reduced_coll_integral` or `k_coll`?).
Since we decided that at the fist stage we only want to report rate violators, but not crash RMG, I decided to capture errors here and `pass` on them. This is just to be on the safe side and not crushing a job near its end for a reason we're not certain of.